content/docs/1.5/scalers/prometheus.md

```diff
@@ -4,7 +4,7 @@ layout = "scaler"
 availability = "v1.0+"
 maintainer = "Community"
 description = "Scale applications based on Prometheus."
-go_file = "prometheus_scaler"
+go_file = "prometheus"
```

Comment:
Would you mind updating our doc in v1.4 & v2.0 as well please?

Reply:
Sure. No problem.

Reply:
Done.